src/elements/EditableTable.tsx

```diff
@@ -0,0 +1,262 @@
+                        return (
+                            <TableRow hover key={index}>
+                                {includeFooter && <TableCell />}
+
+                                {cols.map((col, index) => {
+                                    const value = row[col.name];
+                                    const formattedValue = col.format ? col.format(value) : value;
+                                    return (
+                                        <TableCell
+                                            align={col.align}
+                                            key={index}
+                                            onMouseEnter={(event) =>
+                                                handleMouseEnter(event, row, col.name)
+                                            }
+                                            onMouseLeave={(event) =>
+                                                handleMouseLeave(event, row, col.name)
+                                            }
+                                        >
+                                            {col.editable ? (
+                                                <Input
+                                                    readOnly={false}
+                                                    type={col.type}
+                                                    value={formattedValue}
+                                                    onChange={(event) =>
+                                                        handleCellChange(event, row, col)
+                                                    }
+                                                    inputProps={col.props}
+                                                />
+                                            ) : (
+                                                formattedValue
+                                            )}
+                                        </TableCell>
+                                    );
+                                })}
+                            </TableRow>
+                        );
+                    })}
+                </TableBody>
+                {includeFooter && (
+                    <TableFooter>
+                        <TableRow hover>
+                            <TableCell align="center" sx={TableCellStyleOverrides}>
+                                {footerLabel}
+                            </TableCell>
+                            {cols.map((col, index) =>
+                                footerRow[col.name] ? (
+                                    <TableCell
+                                        align={col.align}
+                                        key={index}
+                                        sx={TableCellStyleOverrides}
+                                    >
+                                        {col.format
+                                            ? col.format(footerRow[col.name])
```

Comment:
I don't think the Column `format` will make sense on the footer in all cases. At least not reliably for custom format functions.

Reply:
I could add a footer format function as a parameter for the column.  I could apply that if it's present and fall back on the data format for other cases.  I think it would be odd not to format the footer.